classes/class3/homework3/routes/cats.js

```diff
@@ -0,0 +1,97 @@
+var mongoose = require('mongoose');
+
+//Define the properties of our cats
+var catSchema = mongoose.Schema({
+	name: String, age: Number, colors: [String]
+});
+var Cat = mongoose.model('Cat', catSchema);
+
+//These are the possible names
+var names = ["Whiskers", "Skittles", "Spots", "Tinky", "Sophie", "Agatha", "Dorothy", "Taz", "Oreo", "Muffin"];
+
+//These are the possible colors
+var colors = ["black", "white", "tabby", "orange", "golden"];
+
+var catList = function(req, res) {
+	//List all of the cats currently stored in the database
+	Cat.find({}, function(err, catList) {
+		if (err) {
+			console.error("Couldn't find cat list: ", err);
```

Comment:
You check for errors, which is great, but then you still try to render the cat list even if there is an error. Right under your log, you want something like       res.status(500).send('Something broke!');